submit/templates/submit/license.html

```diff
@@ -5,28 +5,29 @@
 {% block within_content %}
 <h2 class="is-size-4">Select a license</h2>
 
-<form method="GET" action="{{ url_for('ui.license',submission_id=submission_id) }}">
-<div class="field">
-{% for subfield in form.license %}
-  <div class="control">
-    <label class="radio">
-    {{subfield}}
-    {% if subfield.data %}
-        <a href="{{subfield.data}}">{{subfield.label}}</a>
-    {% else %}
-        {{subfield.label}}
-    {% endif %}
-    </label>
-  </div>
-{% endfor %}
-</div>
+<form method="POST" action="{{ url_for('ui.license',submission_id=submission_id) }}">
+  <div class="field">
+  {% for subfield in form.license %}
+    <div class="control">
 
-{% if form.license.errors %}
-  {% for error in form.license.errors %}
-    <p class="help is-warning">{{ error }}</p>
+      <label class="radio">
+        {{subfield}}
+      {% if subfield.data %}
+          <a href="{{subfield.data}}">{{ subfield.label.text }}</a>
+      {% else %}
+          {{subfield.label}}
```

Comment:
This has to be {{subfield.label.text}} as well, to prevent double label tags (flagged on a11y checker).

Reply:
Fixed